src/plugins/rollbar.js

```diff
@@ -36,10 +36,20 @@ const rollbar = new Rollbar(config);
 export default {
   install(app) {
     app.config.errorHandler = (error, vm, info) => {
-      rollbar.error(error, { vueComponent: vm, info });
```

Comment:
`{ vueComponent: vm }` was the root cause of the crash. Despite literally coming straight out of the [docs](https://docs.rollbar.com/docs/vue-js), attempting to serialize the entire component instance would consistently cause rollbar to crash the entire application with an out of memory error.

Reply:
See this issue in rollbar: https://github.com/rollbar/rollbar.js/issues/1126#issuecomment-1829327874